_includes/sub-views/community/contribute.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## How to contribute
+
+There is multiple ways you can contribute to the project.
+And help is always welcome!
+
+#### Issue Tracker
```

Comment:
it's a nitpick, can we change this header tag to `###`? `####` font size is not much different with a plain text ㅜ_ㅜ

The other titles too :)